cmd/kubeadm/app/discovery/token/token.go

```diff
@@ -27,44 +34,81 @@ import (
 	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
 	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
 	kubeconfigutil "k8s.io/kubernetes/cmd/kubeadm/app/util/kubeconfig"
+	"k8s.io/kubernetes/cmd/kubeadm/app/util/pubkeypin"
 	tokenutil "k8s.io/kubernetes/cmd/kubeadm/app/util/token"
 	bootstrapapi "k8s.io/kubernetes/pkg/bootstrap/api"
 	"k8s.io/kubernetes/pkg/controller/bootstrap"
 )
 
 const BootstrapUser = "token-bootstrap-client"
 
+var (
```

Comment:
Does kubeadm normally declare errors like this? I'd expect the fmt.Errorf call to just be inlined. e.g. https://github.com/kubernetes/kubernetes/blob/93b144c426385b3680cb608ba882141a1ec85841/cmd/kubeadm/app/util/config/masterconfig.go#L58